src/codeu/chat/client/android/app/src/main/java/com/google/codeu/chatme/view/CreateChatActivity.java

```diff
@@ -1,16 +0,0 @@
-package com.google.codeu.chatme.view;
-
-import android.os.Bundle;
-import android.app.Activity;
-import com.google.codeu.chatme.R;
-
-public class CreateChatActivity extends Activity {
```

Comment:
I think this should remain  separate from the `UsersFragment` since the idea would be to select multiple users (by clicking on their respective fragments in a list of fragments) and then clicking a button that would then create a chat that includes all of the selected users.